docs/administration/funding-programs.md

```diff
@@ -6,27 +6,24 @@ This section describes necessary administrative tasks besides the initial [insta
 
 A funding program contains basic data such as title, funding period, application period, budget etc. It is possible to add user-defined properties. By defining a [funding case type](../administration/necessary-adjustments.md) you can influence the funding logic as described [here](../application-states.md) as well as the forms that are visible to applicants in the funding portal.
 
-### Crate a new Funding Program
+### Create a new Funding Program
 
 Navigate to **Funding > Add Funding Program**. Afterward you can enter the following basic values of the program:
 
 * **Title**: The title of the funding program. It is possible to add a long title here.
-*  **Abbreviation**: A short abbreviation of the program title.
+* **Abbreviation**: A short abbreviation of the program title.
 * **Identifier Prefix**: The identifier of a funding case within this funding program will start with the given prefix.
 * **Start Date**: The start date of the drawdown phase.
 * **End Date**: The end date of the drawdown phase.
 * **Requests Start Date**: The start date of the application phase.
 * **Requests End Date**: The end date of the application phase.
 * **Currency**: The currency for the budget.
 * **Budget**: The amount of money that the giving organisation can distribute across the various funding requests.
-* **Funding Case Type**: The type of this funding program. The available case types are
-    * Sonstige Aktivität (AVK1)
-    * Internationale Jugendbegegnung
-    * Sammelantrag Kurs
+* **Funding Case Type**: The funding case type used for applications.
 
 The dates **Requests Start Date** and **Requests End Date** refer to the [application phase](../usage/usage.md#phases-of-a-funding-case). The dates **Start** and **End** refer to the [drawdown phase](../usage/usage.md#phases-of-a-funding-case). In most situations, the application phase ends before the drawdown phase starts, i.e. `Requests Start Date < Requests End Date < Start Date < End Date`.
 
-After creating the funding program, you will be redirected to a page that allows you to take further actions. It is recommended to [edit the funding program permissions](../administration/user-permissions.md#funding-program) via **Actions > Edit permissions** before closing this page. Otherwise, you might not have enough permissions and not find the newly created funding program in the funding program overview.
+After creating the funding program, you will be redirected to a page that allows you to take further actions. It is recommended to edit the possible recipients, the [funding program permissions](../administration/user-permissions.md#funding-program), and the initial funding case permissions, and the  before closing this page.
 
```

Comment:
There are too many commas and "and the" in this sentence.

Reply:
Fixed.